### ISSUE TITLE
dependency-/packaging-issues

### ISSUE THREAD

**Issue by BuckarooBanzay** [Neuromancer56/AsunaPlus#1]
`depends.txt` and `init.lua` should not exist in the root of a modpack (the init.lua file is inert anyway)

and the `depends` field in the `modpack.conf` is invalid (not used) see: https://github.com/minetest/minetest/blob/master/doc/lua_api.md#modpacks

Do i understand your issues correctly: you are trying to create a modpack but minetest trips on the "missing" `asuna_plus_depends` package, right?

**Comment by Neuromancer56** [comment]
No, my issue is that ContentDb will not approve asuna_plus until asuna_plus_depends is first approved on ContentDb.  The game works when I download it just fine.  There is no problem with my code (at least not one that hurts gameplay).  The problem is I need asuna_plus_depends approved on ContentDb.
